io-engine/src/bin/io-engine-client/v1/replica_cli.rs

```diff
@@ -412,6 +427,41 @@ async fn replica_unshare(
     Ok(())
 }
 
+async fn replica_resize(
+    mut ctx: Context,
+    matches: &ArgMatches,
+) -> crate::Result<()> {
+    let uuid = matches.get_one::<String>("uuid").unwrap().to_owned();
+
+    let requested_size =
+        parse_size(matches.get_one::<String>("size").ok_or_else(|| {
```

Comment:
can probably unwrap here same as above since size is a required parameter?

Reply:
Followed from `create` so we can return a clear error :)

Reply:
So the point is what would one like to see let's say when issuing below wrong command:
> ./io-engine-client replica resize f3eb2bd1-e328-4955-8804-521cc28f738f 170MiiB

This:
> thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "170MiiB"', io-engine/src/bin/io-engine-client/v1/replica_cli.rs:444:81
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Aborted (core dumped)

OR
> Error: GrpcStatus { source: Status { code: InvalidArgument, message: "Bad size '170MiiB'", source: None }, backtrace: Backtrace(()) }

Reply:
AIUI the first cannot happen, because the argument is marked as required?
This is why we should be using derive api rather than the builder, then it'd be much clearer.